### DISCUSSION TITLE
Potentially building on VIMuRe

### DISCUSSION THREAD

**Discussion by ChristopherBouma** [latentnetworks/vimure#91] [Q&A]
Hello!
I'm Christopher Bouma, a Computer Science Masters Students from Utrecht University in the Netherlands, and have become very interested in network science, specifically with regard to self-reported social network data. I thoroughly enjoyed the paper by De Bacco et al., and was hoping to do some work in the same area, potentially using VIMuRe. In the discussion of the paper, it's noted that including additional information like covariates on nodes is an area that could be extended! I was wondering whether this is something already being worked on, or whether it's something I could include in my project, and potentially share here.
I'm afraid nothing is set in stone, but it was something that occurred to me as an interesting opportunity, if it's possible!

Regardless, thank you for the work you've done!
Cheers,
Christopher Bouma

**Comment by jonjoncardoso** [comment]
Hi @ChristopherBouma, happy to know that the VIMuRe model and its underlying modelling framework looks useful to you. Yes, please feel free to work on extensions and to share anything you feel relevant here. 

I work most closely with the code itself, so if you end up using or adapting the R/python code, I'll be more than happy to help and give feedback. My colleagues @cdebacco @eapower @danielRedhead might be better suited to comment on our current plans for introducing covariates into the model.
